Parent/Parent/Dashboard/Calendar/CalendarContainerViewController.swift

```diff
@@ -40,15 +40,15 @@ class CalendarContainerViewController: UIViewController {
 
         embed(calendar, in: view, constraintHandler: { child, container in
             child.view.pinToLeftAndRightOfSuperview()
-            child.view.heightAnchor.constraint(equalToConstant: 142).isActive = true
             child.view.topAnchor.constraint(equalTo: container.topAnchor).isActive = true
+            child.view.bottomAnchor.constraint(equalTo: container.bottomAnchor).isActive = true
         })
 
         embed(plannerList, in: view, constraintHandler: { [weak self] child, container in
             guard let ss = self else { return }
             child.view.pinToLeftAndRightOfSuperview()
             NSLayoutConstraint.activate([
-                child.view.topAnchor.constraint(equalTo: ss.calendar.view.bottomAnchor),
+                child.view.topAnchor.constraint(equalTo: ss.calendar.daysContainer.bottomAnchor),
```

Comment:
nice. Does this make the expand/collapse work better?

Reply:
it works, it needs some animation improvements